### PR TITLE
HBW-333 Dynamic delete_if and disable_if options for datetime field

### PR DESCRIPTION
**Pull request by sergeichukd**
Depends on https://github.com/latera/homs/pull/590

### CODE REVIEW

File: spec/support/helpers/hbw_fields_helper.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module Features
+  module HBWFieldsHelper
```
Comment (TimAle) [inline spec/support/helpers/hbw_fields_helper.rb:2]: This datetime picker is used in HOMS as well. Also, let it be FillHelper or DateTimeHelper.

File: spec/hbw/features/bp_form/dynamic_conditions_by_datetime_spec.rb [hunk 2]
```diff
@@ -0,0 +1,118 @@
+feature 'Control fields with dynamic conditions by datetime', js: true do
+  let(:neutral_option_cell) { find(:xpath, "//*[text() = 'Neutral option']") }
+  let(:disable_fields_cell) { find(:xpath, "//*[text() = 'Disable fields']") }
+  let(:hide_fields_cell)    { find(:xpath, "//*[text() = 'Hide fields']") }
```
Comment (TimAle) [inline spec/hbw/features/bp_form/dynamic_conditions_by_datetime_spec.rb:4]: Use `text:` or `exact_text:` option.

**Review by TimAle — CHANGES_REQUESTED** [review]

File: spec/support/helpers/hbw_fields_helper.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+    def fill_in_datetime_field(name, with: '')
+      datetime_field = fill_in(name, with: with)
```
[the remark this answers is not in the document]
Comment (sergeichukd) [reply spec/support/helpers/hbw_fields_helper.rb]: Done

File: spec/support/helpers/hbw_fields_helper.rb (hunk 1 above)
Comment (sergeichukd) [reply spec/support/helpers/hbw_fields_helper.rb]: Done

File: spec/hbw/features/bp_form/dynamic_conditions_by_datetime_spec.rb (hunk 2 above)
Comment (sergeichukd) [reply spec/hbw/features/bp_form/dynamic_conditions_by_datetime_spec.rb]: This lazy variables are redundant (because of copy-past). I will saw off them.

**Review by TimAle — APPROVED** [review]

**Pull request merged**